src/app/pages/tutorial/tutorial.page.ts

```diff
@@ -9,9 +9,21 @@ import { OnboardingService } from '../../services/onboarding/onboarding.service'
   styleUrls: ['./tutorial.page.scss'],
 })
 export class TutorialPage implements OnInit {
+  slideOptsOne = {
```

Comment:
Unnecessary options as they are the default values according to the docs: https://swiperjs.com/api/.